app/src/main/java/com/example/trainingintive/MainActivity.kt

```diff
@@ -2,10 +2,21 @@ package com.example.trainingintive
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.findNavController
+import androidx.navigation.ui.setupWithNavController
+import com.google.android.material.bottomnavigation.BottomNavigationView
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
     }
+
+    override fun onResume() {
+        super.onResume()
+        val bottomNavigationView = findViewById<BottomNavigationView
+            >(R.id.bottom_navigation_view)
+        val navController = findNavController(R.id.nav_host_fragment)
+        bottomNavigationView.setupWithNavController(navController)
```

Comment:
Is there a reason for calling it in `onResume`? It is not working when you call this in `onCreate`?

Reply:
exactly, when it was called in onCreate there was an error telling that MainActivity doesn't have NavController

Reply:
It is known issue – you can read more about this here https://stackoverflow.com/questions/58703451/fragmentcontainerview-as-navhostfragment. It will be better to call this in `onCreate`. You can do this that way: https://developer.android.com/guide/navigation/navigation-getting-started#navigate